cmd/internal/find/primary/meta.go

```diff
@@ -59,8 +59,12 @@ StringPredicate     => [^-].*
 N                   => \d+ (i.e. some number > 0)
 */
 //nolint
-var metaPrimary = Parser.add(&primary{
-    tokens: []string{"-meta", "-m"},
+var metaPrimary = Parser.add(&Primary{
+    Description: "Returns true if the entry's meta attribute satisfies metaP",
```

Comment:
It's not very clear to me what `metaP`'s supposed to stand for.

Reply:
Same with the others, sizeP and timeP.

Reply:
Was thinking `metaP` => `metaPredicate`, and similar thing for sizeP and timeP. I tried to find a concise way of expressing what you specify for them without going into too much detail on how to specify that (which is what `-h <primary>` is meant to do).